frontend/src/components/taste/TasteList.vue

```diff
@@ -73,10 +94,17 @@ export default {
     },
     computed: {
         ...mapState('taste', ['allTastes', 'myTastes']),
-        ...mapGetters('taste', ['getMyTastesIds'])
+        ...mapGetters('taste', ['getMyTastesIds', 'getAllTastesIds', 'getTasteByName'])
     },
     created() {
-        this.$store.dispatch('taste/fetchTastes');
+        this.$store.dispatch('taste/fetchTastes').then(() => {
+            let customTastes = [];
+            this.getAllTastesIds.forEach((id) => {
+                if (this.allTastes.byId[id].user_id)
```

Comment:
this.allTastes.byId[id].user_id should be equal to current user's id?
always use if (this.allTastes.byId[id].user_id) `{...}`

Reply:
Yes, it should be, api returns custom tastes only for current user, so we don`t need to check. But now we can use is_default to check taste is custom or not